main.py

```diff
@@ -1,10 +1,42 @@
 import init_django_orm  # noqa: F401
+import json
 
 from db.models import Race, Skill, Player, Guild
 
 
 def main() -> None:
-    pass
+    with open("players.json") as source:
+        data = json.load(source)
+
+    for players_name, info in data.items():
+        race = info["race"]
+        race_obj, _ = Race.objects.get_or_create(
+            name=race["name"],
+            description=race["description"]
+        )
+
+        skills = info["race"]["skills"]
+        for skill in skills:
+            Skill.objects.get_or_create(
+                name=skill["name"],
+                bonus=skill["bonus"],
+                race=race_obj
+            )
+
+        guild = info["guild"]
```

Comment:
As a result inside player must be a guild or None, not dict
If we have guild equal to dict it will be so bad